### ISSUE TITLE
Open the code for this plugin since this will not be maintained anymore

### ISSUE THREAD

**Issue by pimentelleo** [Drawethree/PRO-DeathChest#27]
Please allow the community to work on it

**Comment by TheSamDickey** [comment]
I agree. This looks like a great plugin. Though without being able to see the code I have no way of trusting it.  There could easily be malware without seeing what's going on under the hood.

I'll happily use this plugin if it becomes open source.
